### ISSUE TITLE
Images are not shown in Word (docx)

### ISSUE THREAD

**Issue by miltondp** [manubot/manubot#364]
If I generate a Word file (docx) for the manuscript (`BUILD_DOCX=true`) using the latest version of manubot and roostock, none of the images are shown (Twitter, Mastodon, figures for the manuscript, etc). This used to work fine before.

![image](https://user-images.githubusercontent.com/172687/222802662-47e3dba4-24cd-4b84-a2d5-d256a7b42656.png)

**Comment by miltondp** [comment]
If it helps, I'm [attaching the Word document](https://github.com/manubot/manubot/files/10885307/manuscript.docx) for the manuscript I'm working on.

**Comment by agitter** [comment]
Is this only affecting svg images? There are known issues with svg images in docx outputs with some prior discussion in https://github.com/manubot/rootstock/pull/40 and https://github.com/greenelab/covid19-review/pull/848#issuecomment-776147608

**Comment by miltondp** [comment]
Ah, that's super helpful, @agitter. By reading those, I just realized that I need to have the command `rsvg-convert` in my system. I don't think I saw that in the documentation, only in issues and discussions in other repos. I wonder if we should add `librsvg` (which provides this command) to `build/environment.yml` in [manubot/rootstock](https://github.com/manubot/rootstock/).

**Comment by agitter** [comment]
I agree it could be a good idea to add `librsvg` to the environment now. What do you think @dhimmel? Exporting to docx with svg images isn't the most common workflow, but it is common enough. We must have had users report issues with this at least 5 times. Who knows how many times it has cause problems that went unreported.

I believe we didn't include `librsvg` in the environment originally because it was [not available for Windows](https://anaconda.org/conda-forge/librsvg/files?version=2.40.19). However, the [latest version](https://anaconda.org/conda-forge/librsvg) is through conda-forge.

Let's move this issue and discussion to the rootstock repo.
